server/tests/services/pageLog.test.js

```diff
@@ -0,0 +1,78 @@
+import { actionTypes } from '../../models/ticket';
+import { fixtures, build } from '../../utils/factories';
+import pageLogService from '../../services/pageLog';
+
+describe('## PageLogService', () => {
+  const timestamp = new Date().getTime();
+  let ticketId;
+
+  beforeEach((done) => {
+    build('ticket', fixtures.ticket())
+      .then((ticket) => {
+        ticketId = ticket.id;
+        done();
+      });
+  });
+
+  describe('# addAction()', () => {
+    context('with valid input parameters', () => {
+      it('adds an action successfully', (done) => {
+        pageLogService.addAction(ticketId, actionTypes.CREATED, timestamp)
```

Comment:
As mentioned above, it's because they are optional, but I should put it in for testing purposes.